build-cli.sh

```diff
@@ -25,25 +25,4 @@ echo Building for Windows...
 CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build -a -ldflags "-X main.cliVersion=$VERSION -extldflags=\"-static\"" -o output/seed-windows-amd64
```

Comment:
Maybe I'm missing something here, but I only see this building for Windows...

Reply:
It's building all three, I just removed the example images part of the build... not sure where you're getting that

Reply:
😢 I totally missed the boat. This is good to go.